app/assets/stylesheets/sticky-footer.scss

```diff
@@ -10,7 +10,7 @@ body {
 }
 
 .page-footer {
-  background-color: $gray-lighter;
+  background-color: #eeeeee;
```

Comment:
(Suggestion only.) You could update the variable to keep all the colours in one place.